include/linux/df-detection.h

```diff
@@ -0,0 +1,44 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+
+#ifndef __LINUX_DOUBLE_FETCH_DETECTION_H__
+#define __LINUX_DOUBLE_FETCH_DETECTION_H__
+
+#include <linux/types.h>
+#include <linux/stackdepot.h>
+
+#ifdef CONFIG_DF_DETECTION
+
+#define DFETCH_INIT_SIZE 16
+#define DFETCH_MAX_RECORDS 1024
+#define MAX_LEN 1 << 20
+#define STACK_DEPTH 64
+#define DFETCH_ENABLE           _IO('c', 254)
+#define DFETCH_DISABLE          _IO('c', 255)
+#define BYTE_MAX 256
+
+/*For each address range used by copy functions*/
```

Comment:
Mind the punctuation in the comments.
There should be spaces after `/*` and before `*/`. Sentences should end with a period.